documentation/createmodal-order.md

```
@@ -0,0 +1,237 @@
+
+> [!IMPORTANT]  
+> 'Specific fields' are fields that aren't numbered and have parenthesis in the beginning. These are fields that are specific to only one model which is indicated in parenthesis.
+>
+> E.g. "(assets) Security objectives"
+
+> [!NOTE]  
+> Categories are only here to differentiate each block of fields, it is not needed to show them on Forms.
+
+##  General rules
+
+#### Unique & frequents
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)